apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import { defineProps, defineEmits } from 'vue';
+
+import { PButtonModal, PI } from '@spaceone/design-system';
+
+const props = defineProps<{
+    visible?: boolean;
+    manageDisabled?: boolean;
+}>();
+
+const emit = defineEmits<{(e: 'update:visible', value: boolean): void;
+    (e: 'confirm'): void;
+}>();
+
+const handleVisibleUpdate = (value: boolean) => {
+    emit('update:visible', value);
+};
+
+const handleConfirm = () => {
+    emit('update:visible', false);
+    emit('confirm');
+};
+
+</script>
+
+<template>
```

Comment:
question: Why is the template below here?

Reply:
It seems default order for Vue3.
My IDE always make vue file like this.